recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -0,0 +1,31 @@
+# Avoid redundant runtime dependency on python3-core
+RDEPENDS:${PN}:remove:class-target = " ${PYTHON_PN}-core" 
+
+require xen-source.inc
+
+FILES:${PN}-test = "\
+    ${libdir}/xen/bin/test-xenstore \
+    ${libdir}/xen/bin/test-resource \
+    ${libdir}/xen/bin/test-paging-mempool\
+"
+
+do_install:append() {
+    rm -f ${D}/${libdir}/xen/bin/init-dom0less
+    rm -f ${D}/${systemd_unitdir}/system/var-lib-xenstored.mount
+    rm -rf ${D}/var
+}
```

Comment:
What is this? why?

Reply:
This recipe taken from one of our repositories with no changes

Reply:
then why have you added this recipe? What is the purpose?

Reply:
Removed